modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/editor/configuration/CKEditorConfigContributor.java

```diff
@@ -81,7 +78,8 @@ public void populateConfigJSONObject(
 			"entities", Boolean.FALSE
 		);
 
-		String extraPlugins = "a11yhelpbtn,itemselector,lfrpopup,media";
+		String extraPlugins = "autogrow,itemselector,lfrpopup,media," +
+			"stylescombo,videoembed";
```

Comment:
Sneaky plugin changes. Was dropping `a11yhelpbtn` intentional?

Reply:
Caught me! 😂 

![sneaky](https://user-images.githubusercontent.com/905006/89185679-ea408c00-d59a-11ea-93c3-d12c0b4810d7.gif)

It was indeed intended. We have some review tickets to figure out how to better do this than the current approach which was already labeled as problematic by UX